### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by BlazeVertex** [boranblok/nntpPoster#76]
How do i say to nntpPoster to post this file (transformers intro song) for exemple And where do i put the news groups i dont see it in the config file 

and i dont get to what to do with this bcs i dint even selected something to post bcs i dont understand it all a video would help tbh 

FileSystemWatcher started
Autoposter started
Notifier started
Verifier started
DB Cleaner started
Press the "s" key to stop after the current operations have finished.

i'm nothing with this info it's just not clear in your readme thing. WHERE DO I SPECEFY MY GROUP AND LOCATION OF WHAT I WANT TO POST?

**Comment by ghost** [comment]
Shouting isn't gonna help you.

**Comment by BlazeVertex** [comment]
> Shouting isn't gonna help you.

You are right i had a rough day and the other week was a big bummer but could you help me?

**Comment by ghost** [comment]
https://github.com/boranblok/nntpPoster/wiki/Watchfolder-Configuration

**Comment by BlazeVertex** [comment]
Good and wel but i my self dont get wiser from this information can you make an exemple video of an step by stem clear guide on how to post i have also autism so i dont really understand bcs there are no tutorials on the web to use this commandline aka program.

**Comment by boranblok** [comment]
I am sorry if things are confusing, writing documentation is not my strong point.

However you might be confusing two parts of the solution. There is a commandline to post one file, and there is a commandline app intended to be run as a service that continuously monitors folders for files and "mirrors" these to usenet. (The main use case there is bulk uploads from other sources or mirroring data)

What do you exactly want to achieve, upload one file, or setup a permanent filewatcher to upload files?

**Comment by BlazeVertex** [comment]
Just one file? OK i understand that you can have a service so that you can basacaly put anything there and then it posts it if i'm correct

So i'm still wondering on the part where to say the prohram Transformersintro.mp4 for example i want that on an Usenet aka on the index so how do i start this (or better said what to do first.  I gave also the info needed in the config file so i dont think i need to do anything there anymore.  Or do i for putting newsgroups and other stuff that i overlooked so idk tbh if you could help me that would be great and again sorry about my first interaction.

**Comment by boranblok** [comment]
Honestly, seeing the scope of what you want to achieve and your technical know-how I would recommend using a usenet uploader with a graphical user interface.

There are several mentioned on the /r/usenet wiki: https://github.com/animetosho/Nyuu/wiki/Usenet-Uploaders

ngpost and EasyUSENET Uploader are really easy to use
